src/main/java/lotto/ui/InputView.java

```diff
@@ -19,4 +30,16 @@ public static int getBonusNo() {
         System.out.println("보너스 볼을 입력해 주세요.");
         return Integer.parseInt(scanner.nextLine());
     }
+
+    public static List<String> getManualLottoNumbers(int manualSelectCount) {
+        if (manualSelectCount == 0) {
+            return null;
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
            return Collections.emptyList();
```